### ISSUE TITLE
Tree accessibility issues

### ISSUE THREAD

**Issue by hisuwh** [primefaces/primevue#7899]
### Describe the bug

We are building our own internal component library using primevue as a base.  One of the reasons we chose primevue was the accessibility baked in.  Generally we have found this to be really good but we have a couple of errors being reported by our environment.

We are using Storybook and the axe-playwright plugin to check accessibility.  We are seeing the following failures reported:

## Tree

<img width="914" height="289" alt="Image" src="https://github.com/user-attachments/assets/e5433bb0-5085-4ac2-a30e-69f934dfb34b" />

<img width="917" height="459" alt="Image" src="https://github.com/user-attachments/assets/b978c70c-c141-438a-bcb3-c91f9888740e" />

In stackblitz repro using Accessibility Insights for Web browser extension:

<img width="1193" height="394" alt="Image" src="https://github.com/user-attachments/assets/d863ca63-f0e1-4b88-af50-8a0f22d06909" />

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-inu6e1i2?file=src%2FApp.vue

### Environment

storybook: 8.5.8
axe-playwright: 2.0.2

### Vue version

3.4.37

### PrimeVue version

4.3.6

### Node version

18.18.1

### Steps to reproduce the behavior

1. Setup storybook environment
2. Add primevue
3. Add axe-playwright plugin and test run
4. Add stories for multiselect and tree components
5. Run accessibility tests

OR

1. Open provided stackblitz
2. Select Accessibility Insights for Web browser extension
3. Select FastPass

### Expected behavior

No accessibility errors

**Comment by hisuwh** [comment]
@tugcekucukoglu the fix in #7904 does not resolve this part of the issue:

<img width="914" height="289" alt="Image" src="https://github.com/user-attachments/assets/03c4a0ad-d434-425a-a383-10d25ce775dc" />

**Comment by hisuwh** [comment]
Updated stackblitz: https://stackblitz.com/edit/primevue-4-vite-issue-template-ubmqqhat?file=package.json

<img width="1890" height="576" alt="Image" src="https://github.com/user-attachments/assets/c46fbf27-4020-445b-90d8-74358b92a710" />
